schedule/yast/maintenance/default_sle15sp3.yaml

```diff
@@ -0,0 +1,53 @@
+---
+# Default ordered sequence of steps for sle15sp3 maintenance update
+bootloader:
+  - installation/bootloader_start
+setup_libyui:
+  - installation/setup_libyui
+product_selection:
+  - installation/product_selection/install_SLES
+license_agreement:
+  - installation/licensing/accept_license
+registration:
+  - installation/add_on_product_installation/add_additional_products
+  - installation/registration/register_via_scc
+extension_module_selection:
+  - installation/module_registration/register_extensions_and_modules
+add_on_product:
+  - installation/add_on_product/skip_install_addons
+add_on_product_installation: []
+system_role: []
```

Comment:
if we use `register_extensions_and_modules` which I think it makes sense, we are planning to add most of the modules and their respective unreleased maintenance updates repos, thus system role should be gnome and we overwrite when this change, like in transactional for example.